package.json

```diff
@@ -0,0 +1,56 @@
+{
+	"name": "trader",
```

Comment:
This feels too generally named, not a issue specific with this PR, but do you think this project needs a better name?

Reply:
I don't think this matters too much here because this name here is used for making an npm package but yeah we can probably find a better name for the app (might take some work to refactor this though haha)

Reply:
renamed the app to agritrader